README.md

```diff
@@ -8,15 +8,17 @@ The yotta Docker image
 
 [Docker](https://www.docker.com/)
 
-### Build the docker image
-
-The helper script ```docker-build``` will create your image. You can pass any extra docker commands into the helper such as ```--no-cache```.
+### TL;DR
 
 ```
-./docker-build.sh <docker-commands>
+wget https://raw.githubusercontent.com/ARMmbed/yotta-docker/master/yotta.sh -O - | sudo tee /usr/local/bin/yotta && sudo chmod +x /usr/local/bin/yotta
```

Comment:
Not sure this would work on all docker hosts, would it be worth having a quick install for Windows, too?